client/src/components/common/Navbar/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from 'antd';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShoppingCart } from '@fortawesome/free-solid-svg-icons';
+import * as ROUTES from '../../../constants/routes';
+import navLogo from '../../../assets/images/berwaz.png';
+import './style.css';
+
+const NavbarComponent = () => {
+  return (
+    <div className="main-navbar-container">
+      <div className="navbar-container">
+        <div className="nav-left">
+          <Link to={ROUTES.CART_PAGE}>
+            <FontAwesomeIcon
+              className="cart-icon"
+              icon={faShoppingCart}
+              size="lg"
+            />
+          </Link>
+          <Link to={ROUTES.SIGNUP_PAGE} className="nav-link">
```

Comment:
You should use the `isAuth`  to check if user has signed in or not in order to display the appropriate message for him.